src/engine/renderer/tr_init.cpp

```diff
@@ -87,22 +87,23 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 	cvar_t      *r_noMarksOnTrisurfs;
 	cvar_t      *r_lazyShaders;
 
-	cvar_t      *r_ext_occlusion_query;
-	cvar_t      *r_ext_draw_buffers;
-	cvar_t      *r_ext_half_float_pixel;
-	cvar_t      *r_ext_texture_float;
-	cvar_t      *r_ext_texture_integer;
-	cvar_t      *r_ext_texture_rg;
-	cvar_t      *r_ext_texture_filter_anisotropic;
-	cvar_t      *r_ext_gpu_shader4;
-	cvar_t      *r_arb_buffer_storage;
-	cvar_t      *r_arb_map_buffer_range;
-	cvar_t      *r_arb_sync;
-	cvar_t      *r_arb_uniform_buffer_object;
-	cvar_t      *r_arb_texture_gather;
-	cvar_t      *r_arb_gpu_shader5;
-	Cvar::Cvar<bool> r_arb_depth_clamp("r_arb_depth_clamp", "Use GL_ARB_depth_clamp if available", Cvar::NONE, true);
-	cvar_t      *r_arb_compute_shader;
+	Cvar::Cvar<bool> r_ext_occlusion_query( "r_ext_occlusion_query", "Use GL_EXT_occlusion_query if available", Cvar::NONE, true );
```

Comment:
It might be nice to make these all `static` in sdl_glimp.cpp since using them elsewhere is basically always a mistake.

Reply:
Done.